filesystem/resizefs.go

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
This is copied from k8s repository and it is licensed under apache license.
Would you copy the [license file](https://github.com/kubernetes/kubernetes/blob/v1.19.7/LICENSES/LICENSE) into root directory?